src/app/js/fields/wizard/StepValue.js

```diff
@@ -5,26 +5,24 @@ import { connect } from 'react-redux';
 import { change } from 'redux-form';
 
 import Step from './Step';
-import { field as fieldPropTypes } from '../../propTypes';
 import { FIELD_FORM_NAME } from '../';
 
 import StepValueValue from './StepValueValue';
 import StepValueColumn from './StepValueColumn';
 import StepValueConcat from './StepValueConcat';
 import StepValueSubresource from './StepValueSubresource';
 
-export const StepValueComponent = ({ field, handleChange, ...props }) => (
+export const StepValueComponent = ({ handleChange, ...props }) => (
     <Step id="step-value" label="field_wizard_step_value" {...props}>
-        <StepValueValue field={field} onChange={handleChange} />
-        <StepValueColumn field={field} onChange={handleChange} />
-        <StepValueConcat field={field} onChange={handleChange} />
-        <StepValueSubresource field={field} onChange={handleChange} />
+        <StepValueValue onChange={handleChange} />
```

Comment:
You remove a prop without changing components' propTypes, or is the propType field use elsewhere?

Reply:
it was unused props, unset in proptypes